src/index/resolveConfig.ts

```diff
@@ -0,0 +1,16 @@
+import { readFileSync } from "fs";
+import findUp from "find-up";
+import { Config } from "./config";
+
+function resolveTsconfig(cwd: string): Config["tsconfig"] | undefined {
+  const tsconfig = findUp.sync(["tsconfig.json", "jsconfig.json"], { cwd });
+  if (!tsconfig) {
+    return undefined;
+  }
+  return JSON.parse(readFileSync(tsconfig).toString()).compilerOptions as any;
```

Comment:
Might need to check if the config extends from another one?

Reply:
yes, I'll probably leave it for another time though. I know it is not a complete solution but I'd like to try this changes first and see how it goes... perhaps we can track it in an issue to address it in the future

Reply:
sure